### PR TITLE
SCAL-192737 Added Number Formatting in ChartColumn.columnProp of SDK

### PR DESCRIPTION
**Pull request by yashvardhan-n**
Defined Number Formatting types

### CODE REVIEW

File: src/types/common.types.ts [hunk 1]
```diff
@@ -17,6 +17,24 @@ import { VisualPropEditorDefinition } from './visual-prop.types';
...
+    String: string;
+    Boolean: boolean;
+    Int: number;
+    Float: number;
+    FileUpload: any;
+    GUID: string;
+    JSON: {
+        [key: string]: any;
+    };
+    JSONObject: any;
```
Comment (harshmeetTS) [inline src/types/common.types.ts:34]: replace any types with unknown

File: src/types/conditional-formatting.types.ts [hunk 2]
```diff
@@ -3,30 +3,16 @@
...
-    Float: number;
-    FileUpload: any;
-    GUID: string;
-    JSON: {
-        [key: string]: any;
-    };
-    JSONObject: any;
-    Long: any;
-};
+
```
Comment (harshmeetTS) [inline src/types/conditional-formatting.types.ts:15]: remove extra line

File: src/types/number-formatting.types.ts [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+export type CustomFormatConfig = {
+    __typename?: 'CustomFormatConfig';
+    format?: Maybe<Scalars['String']>;
+};
+
+export enum NegativeValueFormat {
+    BracesNodash = 'BRACES_NODASH',
+    PrefixDash = 'PREFIX_DASH',
+    SuffixDash = 'SUFFIX_DASH',
+}
```
Comment (harshmeetTS) [inline src/types/number-formatting.types.ts:48]: define all enums on Top

**Review by harshmeetTS — APPROVED** [review]

File: src/types/number-formatting.types.ts (hunk 3 above)
Comment (yashvardhan-n) [reply src/types/number-formatting.types.ts]: ack

**Pull request merged**